extensions/config-yaml/runtime/src/main/java/io/quarkus/yaml/configuration/runtime/YamlConfigConstants.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.yaml.configuration.runtime;
+
+public class YamlConfigConstants {
+
+    public static final String APPLICATION_YML_FILE = "application.yml";
+    public static final String MICROPROFILE_CONFIG_YML_FILE = "microprofile-config.yml";
```

Comment:
This file should be inside `META-INF`, to be close to the spec: https://github.com/eclipse/microprofile-config/#design